spec/lib/vmdb/gettext/debug_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe Vmdb::Gettext::Debug do
+  include Vmdb::Gettext::Debug
```

Comment:
This line bothers me as it could lead to test contamination / sporadic bugs, but I'm not sure if including here includes it only in the example group or if it's global.  If it's being included globally, we have to find another way to test

One such way might be to create an anonymous class with something like:

```ruby
let(:instance) do
  Class.new do
    include Vmdb::Gettext::Debug
  end.new 
end

subject { instance.send(method, *args) }
```

cc @jrafanie